packages/core/src/extension/resizable-iframe-creator.ts

```diff
@@ -58,9 +58,10 @@ export default class ResizableIframeCreator {
 
                     const iframes = iframeResizer(
                         {
+                            autoResize: false,
                             scrolling: false,
                             sizeWidth: false,
-                            heightCalculationMethod: 'bodyScroll',
+                            heightCalculationMethod: 'bodyOffset',
```

Comment:
According to [iframeResizer doc](https://github.com/davidjbradshaw/iframe-resizer/blob/master/docs/parent_page/options.md#widthcalculationmethod), `bodyOffset` and `taggedElement` are the only two methods that will not introudce screen flicker in some browsers.

Therefore, use `bodyOffset` as the default option.